src/CTA.Rules.Config/Constants.cs

```diff
@@ -98,5 +99,18 @@ public class Constants
         public const string ConnectionStrings = "ConnectionStrings";
 
         public static readonly List<string> SupportedMethodModifiers = new List<string>() { "public", "internal", "protected", "private", "abstract", "extern", "override", "static", "unsafe", "virtual", "async" };
+
+
```

Comment:
Do you think we should use enums for these file names? If we get more templates in the future I would prefer to keep typos from making their way in here and it would also be easier to locate this code by looking all references on the enums when we add new enum types.